supporting_content/dimensions/resourcename.md

```diff
@@ -0,0 +1,33 @@
+# Column: ResourceName
+
+## Example provider mappings
+
+Current column mappings found in available data sets:
+
+| Provider  | Data set                 | Provider column                                                                                                                                                                  |
+|-----------|--------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| AWS       | CUR                      | Not present as a separate column in data. <br>Name available in some cases as a part of line_item_resource_id (in the ARN OR as the id for things like s3 buckets)                  |
+| GCP       | Big Query Billing Export | Not present as a separate column in data. <br>Column resource.global_name seems to have a resource name at the very end of the fully qualified resource name (resource.global_name). |
+| Microsoft | Cost details             | ResourceName                                                                                                                                                                     |
+
+## Example usage scenarios
+
+Current values observed in billing data for various scenarios:
+
+See supporting content for Issue #4 (ResourceId). In some cases, the resource name is provided in the resource id as a part of the fully-qualified name
```

Comment:
Do we want to make a reference to temporary issues in GitHub?

Reply:
I think we should avoid it. Also as its currently formatted it won't link so its unclear what Issue #4 to look at.

Reply:
Used a markdown link to reference the resourceid.md inside the supporting content (will only work after resource id is merged).